src/System.Security.Cryptography.Algorithms/src/System.Security.Cryptography.Algorithms.csproj

```diff
@@ -12,6 +12,7 @@
     <AssemblyName>System.Security.Cryptography.Algorithms</AssemblyName>
     <AssemblyVersion Condition="'$(TargetGroup)'=='net46'">4.0.0.0</AssemblyVersion>
     <AssemblyVersion Condition="'$(TargetGroup)'=='net461' or '$(TargetGroup)' == 'netcore50'">4.1.0.0</AssemblyVersion>
+    <AssemblyVersion Condition="'$(TargetGroup)'=='net463'">4.2.1.0</AssemblyVersion>
```

Comment:
@bartonjs I added this as I saw in @ianhays' 1.0.0 PR that he added that and saw that we build a net463 facade here as well. Is this needed?

Reply:
If that asset gets mapped to anything NuGet ever loads, yes (like if we changed it to net47 in the package, but not the target name).  If it ended up as an abandoned identifier it shouldn't matter.

Facades can't have their assembly version change.  So it's safer to freeze it now than us to have to figure out how to unbreak it if it did matter :smile:.

Reply:
Ok, so I'll leave it frozen like this then :)